modules/set-up-oauth-20-credentials-for-your-gcp-project.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-gsuite-identity.adoc
+:_module-type: PROCEDURE
+[id="set-up-oauth-20-credentials-for-your-gcp-project_{context}"]
+= Setting up OAuth 2.0 credentials for your GCP project
+
+To configure G Suite as an identity provider for {product-title} you must first configure OAuth 2.0 credentials for your GCP project.
+
+.Prerequisites
+* You must have administrator-level access to your organization's G Suite account to create a new project, or permissions to create and configure OAuth 2.0 credentials for an existing project. Red Hat recommends that you create a new project for managing access to {product-title}.
+
+.Procedure
+. Create a new Google Cloud Platform (GCP) project, see the official Google documentation topic https://cloud.google.com/resource-manager/docs/creating-managing-projects[creating and managing projects].
```

Comment:
I don't think that "official" is necessary here

Reply:
Thanks, I've also removed some other instances of this.